recorder.go

```diff
@@ -47,51 +52,79 @@ func NewRecorder(ctx context.Context, opts ...Option) (*Recorder, error) {
 	if err != nil {
 		return nil, err
 	}
-	return &Recorder{
+
+	rec := &Recorder{
 		project:     options.projectID,
 		ctx:         ctx,
 		traceClient: c,
 		log:         options.log,
-	}, nil
+	}
+
+	bundler := bundler.NewBundler((*pb.Trace)(nil), func(bundle interface{}) {
```

Comment:
I am not sure about it, because Recorder implements `basictracer.SpanRecorder` interface, so it used onlt as this interface.